packages/ui/src/components/navbar-new/sidebar-search-new/sidebar-search-new.tsx

```diff
@@ -0,0 +1,56 @@
+import { FormHTMLAttributes, ReactNode } from 'react'
+
+import { Icon } from '@components/icon'
+import { Sidebar } from '@components/sidebar/sidebar'
+
+import { Button, Input, Label } from '@harnessio/ui/components'
+
+import { useSearch } from './search-context'
+
+interface SidebarSearchProps extends FormHTMLAttributes<HTMLFormElement> {
+  logo: ReactNode
+}
+
+export function SidebarSearchNew(props: SidebarSearchProps) {
+  const searchContext = useSearch()
+
+  if (!searchContext) {
+    console.warn('⚠️ Search context is null, returning early.')
+    return null
+  }
+
+  const { setIsOpen } = searchContext
+
+  return (
+    <form {...props}>
+      {props.logo}
+      <Sidebar.Group className="py-0 -mt-2 px-2">
```

Comment:
```suggestion
      <Sidebar.Group className="py-0">
```
1. it seems we can get rid of `-mt-2` which breaks the height of the form parent, if we need to get closer to the logo we can just change its current height

2. We also do not need to specify `px-2`, because initially Sidebar.Group has already set the necessary value for it.

<img width="248" alt="image" src="https://github.com/user-attachments/assets/05eb629f-6f34-4574-b026-af744036342f" />
<img width="926" alt="image" src="https://github.com/user-attachments/assets/8a47a623-fdac-404c-a2b9-979fef890eb0" />